### PR TITLE
feature: ProductRequest class added

### PR DESCRIPTION
**Pull request by dashingSushovan**
#37

### CODE REVIEW

**Review by smruti-banty — COMMENTED** [review]
What about update product. 
- Update case also you have to mention DTO class

File: backend/src/main/java/com/moshop/backend/controller/ProductController.java
Comment (smruti-banty) [inline backend/src/main/java/com/moshop/backend/controller/ProductController.java:1]: Use interface for injection, Don't use its Impl class

Comment (smruti-banty) [inline backend/src/main/java/com/moshop/backend/controller/ProductController.java:1]: For put request response should be ACCEPTED

File: backend/src/main/java/com/moshop/backend/model/dto/ProductResponseDTO.java
Comment (smruti-banty) [inline backend/src/main/java/com/moshop/backend/model/dto/ProductResponseDTO.java:1]: Space Required in between line 8 and 9

**Review by smruti-banty — COMMENTED** [review]
change according to comment

**Review by Abinashkjcjanscinanconanc — APPROVED** [review]
LGTM

**Pull request merged**